actions/arteria_delivery_service.py

```diff
@@ -121,13 +121,17 @@ def parse_stage_order_ids_from_response(response):
 
         return result
 
-    def stage_runfolder(self, runfolder_name, projects):
+    def stage_runfolder(self, runfolder_name, projects,restrict_to_projects):
         stage_runfolder_endpoint = '{}/api/1.0/stage/runfolder/{}'.format(self.delivery_service_location, runfolder_name)
 
-        if projects:
-            payload = {'projects': projects['projects']}
+        if restrict_to_projects == 'None':
+            if projects:
+                payload = {'projects': projects['projects']}
+            else:
+                payload = {}
         else:
-            payload = {}
+            restrict_to_projects_list = restrict_to_projects.split(",")
```

Comment:
Maybe to help sanitize the input a little bit we should first do a `strip` on `restrict_to_projects`, to remove any white spaces that might sneak in? https://www.tutorialspoint.com/python/string_strip.htm